aspnetcore/toc.yml

```
@@ -700,6 +700,14 @@ items:
     items:
       - name: Minimal APIs overview
         uid: fundamentals/minimal-apis
+      - name: WebApplication and WebApplicationBuilder
```

Is there a more scenario-based title we could use here. Something like "Constructing minimal application" or what not...

---

```suggestion
      - name: Constructing Minimal API apps
```

I slightly edited this to match the naming convention of the other sections. I still think "WebApplication and WebApplicationBuilder" is more obvious and clearly defined. "Constructing minimal applications" could literally describe creating the entire application including the routes and middleware.

Does anyone else have an opinion on this?

---

> I still think "WebApplication and WebApplicationBuilder" is more obvious and clearly defined. "Constructing minimal applications" could literally describe creating the entire application including the routes and middleware.

I agree. I'm not fond of "WebApplication and WebApplicationBuilder" as it doesn't describe the content very clearly -- but I think it's better than "Constructing minimal applications" since that could be the title for the entire cheat sheet. WA and WAB could possibly be split up into several H2 sections with titles more clearly describing their contents, like "Working with ports", "Working with certificates" etc. But that's a task for another time, and I would keep WA and WAB for now.